pkgs/development/python-modules/orange3/default.nix

```diff
@@ -0,0 +1,169 @@
+{ lib
+, buildPythonPackage
+, copyDesktopItems
+, fetchurl
+, makeDesktopItem
+, fetchFromGitHub
+, nix-update-script
+, pythonRelaxDepsHook
+, python
+, baycomp
+, bottleneck
+, chardet
+, cython
+, httpx
+, joblib
+, keyring
+, keyrings-alt
+, matplotlib
+, numpy
+, openpyxl
+, opentsne
+, orange-canvas-core
+, orange-widget-base
+, pandas
+, pyqtgraph
+, pyqtwebengine
+, python-louvain
+, pyyaml
+, qt5
+, qtconsole
+, requests
+, scikit-learn
+, scipy
+, sphinx
+, serverfiles
+, xlrd
+, xlsxwriter
+}:
+
+let
+self = buildPythonPackage rec {
+    pname = "orange3";
+    version = "3.35.0";
+    format = "pyproject";
+
+    src = fetchFromGitHub {
+      owner = "biolab";
+      repo = "orange3";
+      rev = "refs/tags/${version}";
+      hash = "sha256-dj/Z4uOjA4nidd45dnHZDyHZP6Fy/MGC8asqOPV7U7A=";
+    };
+
+    pythonRelaxDeps = [
+      "scikit-learn"
+      "pyqtgraph" # TODO: remove after bump with a version greater than 0.13.1
+    ];
+
+    postPatch = ''
```

Comment:
Why do you have both `postPatch` and `pythonRelaxDeps`? I think only one or another is necessary (preferably the `pythobRelaxDeps`).